gcc/testsuite/lib/rust.exp

```diff
@@ -171,7 +171,7 @@ proc rust_target_compile { source dest type options } {
     global individual_timeout
 
     # HACK: guard against infinite loops in the compiler
-    set individual_timeout 10
+    set individual_timeout 300
```

Comment:
Shouldn't we instead revert PR #1107 "tests: Improve timeout handling" commit 997894f6372d057e22cd54cc7b9fefdfdc77520a "testsuite/rust: add a 10-second timeout for each compile test"?

(..., and why is that in this PR here, "testsuite: Handle Windows carriage returns properly"?)  ;-|

Reply:
Yes, sorry about that this was a bit of a testing ground for nightstrike to pull and test on their machine. I'll clean that up and instead open a PR to revert the timeout change